router/eventorder_test.go

```diff
@@ -145,8 +145,6 @@ func TestEventOrderGuarantee(t *testing.T) {
 			config.Set("Router.updateStatusBatchSize", 100)
 			config.Set("Router.maxStatusUpdateWait", "10ms")
 
-			defer config.Reset()
-
```

Comment:
resetting the singleton config instance after the test is done makes sure that overridden config values are reset and won't affect other tests